articles/azure-app-configuration/quickstart-resource-manager.md

```diff
@@ -55,6 +55,13 @@ Two Azure resources are defined in the template:
 >
 > For example, to create a key-value pair with key name `AppName:DbEndpoint` and label name `Test`, the resource name should be `AppName~3ADbEndpoint$Test`.
 
+> [!NOTE]
+> App Configuration allows key-value data access over a [private link](/concept-private-endpoint) from your virtual network (VNET). By default, when the feature is enabled, all requests for your App Configuration data over the public network will be denied. As the ARM template runs outside of your VNET, data access from an ARM template will not be allowed. To allow data access from an ARM template when a private link is used, you can enable public network access using this Azure CLI command, but please take the security implication into consideration.
+>
+>```azurecli-interactive
+>az appconfig update -g MyResourceGroup -n MyAppConfiguration --enable-public-network true
```

Comment:
After running the command, the configuration explorer in Portal is accessible now. But I'm stilling seeing ARM deployment failure. I might have to follow-up with Gary. 
```
Deployment failed. Correlation ID: fbdcf426-9d19-4fb9-b343-5e844e9b043e. {
  "error": {
    "code": "PrivateLinkActive",
    "message": "The operation cannot be performed because private link is active on the resource.",
    "additionalInfo": [
      {
        "type": "ActivityId",
        "info": {
          "activityId": "4f189342-26bf-4e7d-aadb-5f24ae28afec"
        }
      }
    ]
  }
}
```

Reply:
I have a question though. What's the point of having private link after enabling public access? :)